src/components/TypingIndicator/TypingIndicator.js

```diff
@@ -48,9 +49,10 @@ const TypingIndicator = (props) => {
 
 TypingIndicator.propTypes = {
   /** @see See [chat context](https://getstream.github.io/stream-chat-react/#chatcontext) doc */
+  // @ts-ignore
   client: PropTypes.object,
```

Comment:
We want to move this to useContext? Or will it be in different PR?

Reply:
yeah that probably made sense, hadn't figured that out yet when I created this pr

Reply:
this component is not actually using the context

Reply:
just get's passed as props